From ee80efdaa9cd7c1281daf79c57e8e40d9c750e25 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 21 Nov 2021 10:50:38 +1100
Subject: [PATCH 1/2] do not try to detect & set build number

we're only building released versions in conda-forge anyway...
---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 459e062..68c3443 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,15 +18,6 @@ option(BUILD_PYTHON "Build python if ON" ON)
 #-------------------------------------------------------------------------------
 SET(BUILD_NUMBER 0)
 SET(VERSION_STRING "${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}")
-find_package(Git)
-if(GIT_FOUND)
-    execute_process(
-            COMMAND ${GIT_EXECUTABLE} describe --match INITIAL_BUILD
-            OUTPUT_VARIABLE DESCRIBE_BUILD
-            OUTPUT_STRIP_TRAILING_WHITESPACE)
-    string(REGEX MATCH "[0-9]+" BUILD_NUMBER ${DESCRIBE_BUILD})
-endif()
-ADD_DEFINITIONS(-DBUILD_NUMBER=${BUILD_NUMBER})
 ADD_DEFINITIONS(-DVERSION_STRING=${VERSION_STRING})
 #-------------------------------------------------------------------------------
 
-- 
2.32.0.windows.2

